### ISSUE TITLE
PHP Notice: Undefined index: host in class.two-factor-fido-u2f.php

### ISSUE THREAD

**Issue by roborourke** [humanmade/altis-security#25]
> Notice: Undefined index: host in /usr/src/app/vendor/humanmade/two-factor/providers/class.two-factor-fido-u2f.php on line 95

Seen when starting local server.

**Comment by roborourke** [comment]
Stack trace:
```
[23-Oct-2019 11:14:20 UTC] PHP Notice:  Undefined index: host in /usr/src/app/vendor/humanmade/two-factor/providers/class.two-factor-fido-u2f.php on line 95
[23-Oct-2019 11:14:20 UTC] PHP Stack trace:
[23-Oct-2019 11:14:20 UTC] PHP   1. {main}() /usr/local/bin/wp:0
[23-Oct-2019 11:14:20 UTC] PHP   2. include() /usr/local/bin/wp:4
[23-Oct-2019 11:14:20 UTC] PHP   3. include() phar:///usr/local/bin/wp/php/boot-phar.php:11
[23-Oct-2019 11:14:20 UTC] PHP   4. WP_CLI\bootstrap() phar:///usr/local/bin/wp/vendor/wp-cli/wp-cli/php/wp-cli.php:23
[23-Oct-2019 11:14:20 UTC] PHP   5. WP_CLI\Bootstrap\LaunchRunner->process() phar:///usr/local/bin/wp/vendor/wp-cli/wp-cli/php/bootstrap.php:74
[23-Oct-2019 11:14:20 UTC] PHP   6. WP_CLI\Runner->start() phar:///usr/local/bin/wp/vendor/wp-cli/wp-cli/php/WP_CLI/Bootstrap/LaunchRunner.php:23
[23-Oct-2019 11:14:20 UTC] PHP   7. WP_CLI\Runner->load_wordpress() phar:///usr/local/bin/wp/vendor/wp-cli/wp-cli/php/WP_CLI/Runner.php:1111
[23-Oct-2019 11:14:20 UTC] PHP   8. require() phar:///usr/local/bin/wp/vendor/wp-cli/wp-cli/php/WP_CLI/Runner.php:1188
[23-Oct-2019 11:14:20 UTC] PHP   9. do_action() /usr/src/app/wordpress/wp-settings.php:525
[23-Oct-2019 11:14:20 UTC] PHP  10. WP_Hook->do_action() /usr/src/app/wordpress/wp-includes/plugin.php:465
[23-Oct-2019 11:14:20 UTC] PHP  11. WP_Hook->apply_filters() /usr/src/app/wordpress/wp-includes/class-wp-hook.php:310
[23-Oct-2019 11:14:20 UTC] PHP  12. Two_Factor_Core::get_providers() /usr/src/app/wordpress/wp-includes/class-wp-hook.php:286
[23-Oct-2019 11:14:20 UTC] PHP  13. Two_Factor_FIDO_U2F::get_instance() /usr/src/app/vendor/humanmade/two-factor/class.two-factor-core.php:121
[23-Oct-2019 11:14:20 UTC] PHP  14. Two_Factor_FIDO_U2F->__construct() /usr/src/app/vendor/humanmade/two-factor/providers/class.two-factor-fido-u2f.php:41
[23-Oct-2019 11:14:20 UTC] PHP  15. Two_Factor_FIDO_U2F::get_u2f_app_id() /usr/src/app/vendor/humanmade/two-factor/providers/class.two-factor-fido-u2f.php:57
```

**Comment by roborourke** [comment]
This is because nothing is returned from `home_url()` on the `init` hook in the CLI install process. Will see if defining `WP_HOME` whenever the `WP_CLI` class exists resolves it.

**Comment by roborourke** [comment]
Fixed in 3.0
